canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html

```diff
@@ -59,7 +59,7 @@ <h4 class="list-group-item-heading">Find Course Info</h4>
         </div>
 
       {% if cross_listing_allowed %}
-        <div class="list-group col-md-4">
+        <div id="id-cross-listing-div" class="list-group col-md-4">
```

Comment:
Style nitpick...don't think we need to prefix this with `id-`.  Maybe put the id on the button, call it `cross-listing-button`?